cx_Freeze/finder.py

```diff
@@ -649,6 +650,13 @@ def ReportMissingModules(self):
                              "may not be needed on this platform.\n")
             sys.stdout.write("\n")
 
+    def SetOptimizeFlag(self, optimizeFlag):
+        """Set a new value of optimize flag and returns the previous value."""
+        previous = self.optimizeFlag
+        if -1 <= optimizeFlag <= 2:
+            self.optimizeFlag = optimizeFlag
+        return previous
```

Comment:
Should an exception be raised for this? Or should it just be passed through and let Python raise the exception when it sees the invalid value?

Reply:
As in general this value is already checked before reaching Freezer.py, and there it was not checked, so I just did a basic check. If it is a valid value, it is accepted, otherwise ignore it.

Reply:
I added a comment.

Reply:
Ok.